Tidal/dist/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+  "author": {
+    "name": "SinecKers /Yasar",
+    "id": "380411952666640385"
+  },
+  "url": [
+    "listen.tidal.com",
+    "tidal.com"
+  ],
+  "description": {
+    "tr": "Tidal yüksek ses kalitesi ve ucuz fiyatlarla hizmet sağlayan bir müzik sitesidir",
+    "en": "Tidal is a music stream service with very high sound quality"
+  },
+  "service": "Tidal",
+  "version": "1.0.0",
+  "logo": "https://who.is-inside.me/pXP8PRGK.jpg",
+  "thumbnail": "https://who.is-inside.me/5IJ3K6fe.png",
+  "color": "#000000",
+  "tags": [
```

Comment:
set a color

Reply:
Changed the icons and the color of service is black ?

Reply:
oh okay nvm then about the color xD

Reply:
so won't you approve xd

Reply:
update the small image keys? Or upload proof that you did so

Reply:
![ntJC0i9j80](https://user-images.githubusercontent.com/26010736/81339868-86034e80-90b7-11ea-8b17-acf3b61861a5.png)
![IiXSthek8S](https://user-images.githubusercontent.com/26010736/81340110-f611d480-90b7-11ea-8b38-3db60142ba90.png)